# Global Tuberclosis Incidence Dataset.


*  Provided by member states and national TB programmes to the World Health Organisation (WHO).
*   Compiled by WHO.




How do we add the required libraries to our working environment?

We introduce our python libraries namely pandas for dataset manipulation and numpy for calculation functions using the "import" function.   

In [32]:
import pandas as pd
import numpy as np

How do we add the dataset to the enviroment?

We access the data set from its storage on my git-hub repository using the pandas.read_csv function from the pandas library

In [47]:
Global_Incidence = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/C288D13_ALL_LATEST.csv')
Australia_Oceania = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/%20Australia_Oceania.csv')
Europe = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/%20Europe.csv')
North_America = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/%20North_America.csv')
South_America = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/%20South_America.csv')
Africa = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/Africa.csv')
Asia = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/Asia.csv')
Global_incidence_by_years = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/%20Global_incidence_by_years.csv')
Global_TB_incidence_grouped_in_regions = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/%20Global_TB_incidence_grouped_in_regions.csv')
Income = pd.read_csv('https://raw.githubusercontent.com/bamundagaaloyzius/MCS_7101-Machine-Learning-/main/Income.csv')

How many data points and features do we have in our Global_Incidence dataset?

The rows represent the data points while the columns represent the features. This makes 4784 data points and 14 features.  

In [8]:
Global_Incidence.shape


(4784, 14)

How many data points and features do we have in our Africa dataset?

The rows represent the data points while the columns represent the features. This makes 1208 data points and 8 features.

In [9]:
Africa.shape

(1208, 8)

How do we distiguish numerical and categorical features?

The .dtypes function from pandas library helps us to note the data types in order to distinguish numerals from Alphabetical letters.

In [10]:
Global_Incidence.dtypes

IND_ID                     object
IND_CODE                   object
IND_UUID                   object
IND_PER_CODE               object
DIM_TIME                    int64
DIM_TIME_TYPE              object
DIM_GEO_CODE_M49            int64
DIM_GEO_CODE_TYPE          object
DIM_PUBLISH_STATE_CODE     object
IND_NAME                   object
GEO_NAME_SHORT             object
RATE_PER_100000_N         float64
RATE_PER_100000_NL        float64
RATE_PER_100000_NU        float64
dtype: object

In [11]:
Africa.dtypes

DIM_TIME                int64
DIM_GEO_CODE_M49        int64
DIM_GEO_CODE_TYPE      object
CONTINENT              object
GEO_NAME_SHORT         object
RATE_PER_100000_N     float64
RATE_PER_100000_NL    float64
RATE_PER_100000_NU    float64
dtype: object

How do we know the features in the dataset?

The list(data.columns) function creates a list of the elements in the first row thus the features.
Lets consider Global_Incidence and Africa datasets below.

In [12]:
list(Global_Incidence.columns)

['IND_ID',
 'IND_CODE',
 'IND_UUID',
 'IND_PER_CODE',
 'DIM_TIME',
 'DIM_TIME_TYPE',
 'DIM_GEO_CODE_M49',
 'DIM_GEO_CODE_TYPE',
 'DIM_PUBLISH_STATE_CODE',
 'IND_NAME',
 'GEO_NAME_SHORT',
 'RATE_PER_100000_N',
 'RATE_PER_100000_NL',
 'RATE_PER_100000_NU']

- **IND_ID** : Indicator unique identifier
- **IND_CODE** : Indicator code
- **IND_UUID** : Indicator code
- **IND_PER_CODE** : Indicator code
- **DIM_TIME** : The year of submission
- **DIM_TIME_TYPE** : The time allocation
- **DIM_GEO_CODE_M49** : The geographical code
- **DIM_GEO_CODE_TYPE** : The geographical entity
- **DIM_PUBLISH_STATE_CODE** : This states whether the data is published by WHO or NOt.
- **IND_NAME**: Disease case for this case Tuberclosis
- **GEO_NAME_SHORT** : country or region of data collection
- **RATE_PER_100000_N** : Average number of incidence cases for every 100,000 people for that year.
- **RATE_PER_100000_NL** : Maximum number of incidence cases for every 100,000 people for that year.
- **RATE_PER_100000_NU** : Minimum number of incidence cases for every 100,000 people for that year.

In [13]:
list(Africa.columns)

['DIM_TIME',
 'DIM_GEO_CODE_M49',
 'DIM_GEO_CODE_TYPE',
 'CONTINENT',
 'GEO_NAME_SHORT',
 'RATE_PER_100000_N',
 'RATE_PER_100000_NL',
 'RATE_PER_100000_NU']

how do we get a summary of the data point to feature relationship?

We can utilise the .head or .tail function to return the first and last data points respectively as arranged in the dataset.

In [23]:
Global_Incidence.head()

IND_ID        IND_CODE IND_UUID    IND_PER_CODE  DIM_TIME  \
0  C288D13MDG_0000000020  MDG_0000000020  C288D13  MDG_0000000020      2002   
1  C288D13MDG_0000000020  MDG_0000000020  C288D13  MDG_0000000020      2010   
2  C288D13MDG_0000000020  MDG_0000000020  C288D13  MDG_0000000020      2010   
3  C288D13MDG_0000000020  MDG_0000000020  C288D13  MDG_0000000020      2004   
4  C288D13MDG_0000000020  MDG_0000000020  C288D13  MDG_0000000020      2005   

  DIM_TIME_TYPE  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE  \
0          YEAR               795               COUNTRY   
1          YEAR               914  WORLDBANKINCOMEGROUP   
2          YEAR               554               COUNTRY   
3          YEAR               598               COUNTRY   
4          YEAR               798               COUNTRY   

  DIM_PUBLISH_STATE_CODE            IND_NAME                 GEO_NAME_SHORT  \
0              PUBLISHED  Tuberculosis cases                   Turkmenistan   
1              PUBLISHED  Tuberculosis cases  Upper-middle-income economies   
2              PUBLISHED  Tuberculosis cases                    New Zealand   
3              PUBLISHED  Tuberculosis cases               Papua New Guinea   
4              PUBLISHED  Tuberculosis cases                         Tuvalu   

   RATE_PER_100000_N  RATE_PER_100000_NL  RATE_PER_100000_NU  
0              100.0                52.0               162.0  
1              120.0               103.0               137.0  
2                8.0                 6.8                 9.2  
3              432.0               280.0               617.0  
4              138.0               129.0               148.0

In [24]:
Africa.tail()

DIM_TIME  DIM_GEO_CODE_M49 DIM_GEO_CODE_TYPE CONTINENT GEO_NAME_SHORT  \
1203      2018               894           COUNTRY    Africa         Zambia   
1204      2019               894           COUNTRY    Africa         Zambia   
1205      2020               894           COUNTRY    Africa         Zambia   
1206      2021               894           COUNTRY    Africa         Zambia   
1207      2022               894           COUNTRY    Africa         Zambia   

      RATE_PER_100000_N  RATE_PER_100000_NL  RATE_PER_100000_NU  
1203              346.0               221.0               500.0  
1204              333.0               216.0               474.0  
1205              319.0               208.0               455.0  
1206              307.0               195.0               443.0  
1207              295.0               184.0               431.0

Incase i wanted to see the trend of the incidence in Uganda from 2000 to 2022, what do i use?

We search through the feature named GEO_NAME_SHORT for the tag Uganda using the .loc function.

In [25]:
Africa.loc[Africa['GEO_NAME_SHORT'] == 'Uganda']

DIM_TIME  DIM_GEO_CODE_M49 DIM_GEO_CODE_TYPE CONTINENT GEO_NAME_SHORT  \
1116      2000               800           COUNTRY    Africa         Uganda   
1117      2001               800           COUNTRY    Africa         Uganda   
1118      2002               800           COUNTRY    Africa         Uganda   
1119      2003               800           COUNTRY    Africa         Uganda   
1120      2004               800           COUNTRY    Africa         Uganda   
1121      2005               800           COUNTRY    Africa         Uganda   
1122      2006               800           COUNTRY    Africa         Uganda   
1123      2007               800           COUNTRY    Africa         Uganda   
1124      2008               800           COUNTRY    Africa         Uganda   
1125      2009               800           COUNTRY    Africa         Uganda   
1126      2010               800           COUNTRY    Africa         Uganda   
1127      2011               800           COUNTRY    Africa         Uganda   
1128      2012               800           COUNTRY    Africa         Uganda   
1129      2013               800           COUNTRY    Africa         Uganda   
1130      2014               800           COUNTRY    Africa         Uganda   
1131      2015               800           COUNTRY    Africa         Uganda   
1132      2016               800           COUNTRY    Africa         Uganda   
1133      2017               800           COUNTRY    Africa         Uganda   
1134      2018               800           COUNTRY    Africa         Uganda   
1135      2019               800           COUNTRY    Africa         Uganda   
1136      2020               800           COUNTRY    Africa         Uganda   
1137      2021               800           COUNTRY    Africa         Uganda   
1138      2022               800           COUNTRY    Africa         Uganda   

      RATE_PER_100000_N  RATE_PER_100000_NL  RATE_PER_100000_NU  
1116              276.0               106.0               527.0  
1117              266.0               137.0               437.0  
1118              256.0               150.0               390.0  
1119              248.0               156.0               360.0  
1120              240.0               156.0               341.0  
1121              233.0               145.0               342.0  
1122              227.0               141.0               333.0  
1123              222.0               139.0               324.0  
1124              217.0               139.0               312.0  
1125              213.0               135.0               309.0  
1126              210.0               135.0               301.0  
1127              207.0               141.0               286.0  
1128              205.0               133.0               291.0  
1129              203.0               132.0               289.0  
1130              202.0               127.0               295.0  
1131              202.0               119.0               305.0  
1132              201.0               118.0               307.0  
1133              201.0               118.0               306.0  
1134              200.0               118.0               304.0  
1135              200.0               117.0               303.0  
1136              199.0               120.0               298.0  
1137              199.0               119.0               298.0  
1138              198.0               119.0               297.0

In [26]:
Global_TB_incidence_grouped_in_regions

DIM_TIME  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE Unnamed: 3  \
0        2000               910  WORLDBANKINCOMEGROUP       none   
1        2001               910  WORLDBANKINCOMEGROUP       none   
2        2002               910  WORLDBANKINCOMEGROUP       none   
3        2003               910  WORLDBANKINCOMEGROUP       none   
4        2004               910  WORLDBANKINCOMEGROUP       none   
..        ...               ...                   ...        ...   
225      2018               958             WHOREGION       none   
226      2019               958             WHOREGION       none   
227      2020               958             WHOREGION       none   
228      2021               958             WHOREGION       none   
229      2022               958             WHOREGION       none   

            GEO_NAME_SHORT  RATE_PER_100000_N  RATE_PER_100000_NL  \
0    High-income economies               21.0                20.0   
1    High-income economies               21.0                21.0   
2    High-income economies               21.0                20.0   
3    High-income economies               21.0                20.0   
4    High-income economies               19.0                18.0   
..                     ...                ...                 ...   
225        Western Pacific               96.0                79.0   
226        Western Pacific               94.0                77.0   
227        Western Pacific               93.0                75.0   
228        Western Pacific               94.0                74.0   
229        Western Pacific               96.0                75.0   

     RATE_PER_100000_NU  
0                  22.0  
1                  22.0  
2                  22.0  
3                  22.0  
4                  20.0  
..                  ...  
225               115.0  
226               112.0  
227               113.0  
228               116.0  
229               120.0  

[230 rows x 8 columns]

In [48]:
Income

DIM_TIME  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE  \
0       2000               910  WORLDBANKINCOMEGROUP   
1       2001               910  WORLDBANKINCOMEGROUP   
2       2002               910  WORLDBANKINCOMEGROUP   
3       2003               910  WORLDBANKINCOMEGROUP   
4       2004               910  WORLDBANKINCOMEGROUP   
..       ...               ...                   ...   
87      2018               914  WORLDBANKINCOMEGROUP   
88      2019               914  WORLDBANKINCOMEGROUP   
89      2020               914  WORLDBANKINCOMEGROUP   
90      2021               914  WORLDBANKINCOMEGROUP   
91      2022               914  WORLDBANKINCOMEGROUP   

                   GEO_NAME_SHORT  RATE_PER_100000_N  RATE_PER_100000_NL  \
0           High-income economies               21.0                20.0   
1           High-income economies               21.0                21.0   
2           High-income economies               21.0                20.0   
3           High-income economies               21.0                20.0   
4           High-income economies               19.0                18.0   
..                            ...                ...                 ...   
87  Upper-middle-income economies               95.0                88.0   
88  Upper-middle-income economies               92.0                86.0   
89  Upper-middle-income economies               90.0                84.0   
90  Upper-middle-income economies               91.0                85.0   
91  Upper-middle-income economies               95.0                87.0   

    RATE_PER_100000_NU  
0                 22.0  
1                 22.0  
2                 22.0  
3                 22.0  
4                 20.0  
..                 ...  
87               103.0  
88                99.0  
89                96.0  
90                98.0  
91               102.0  

[92 rows x 7 columns]

What is the incidence trend for Low income countries?

The incidence trend over the period 2000 to 2022 can be acquired by using the .loc function in the GEO_NAME_SHORT column to look for 'Low-income-countries'. The incidence drops from an average of 274 to 181 cases per 100,000 people.

In [43]:
Income.loc[Income['GEO_NAME_SHORT'] == 'Low-income economies']

DIM_TIME  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE        GEO_NAME_SHORT  \
23      2000               911  WORLDBANKINCOMEGROUP  Low-income economies   
24      2001               911  WORLDBANKINCOMEGROUP  Low-income economies   
25      2002               911  WORLDBANKINCOMEGROUP  Low-income economies   
26      2003               911  WORLDBANKINCOMEGROUP  Low-income economies   
27      2004               911  WORLDBANKINCOMEGROUP  Low-income economies   
28      2005               911  WORLDBANKINCOMEGROUP  Low-income economies   
29      2006               911  WORLDBANKINCOMEGROUP  Low-income economies   
30      2007               911  WORLDBANKINCOMEGROUP  Low-income economies   
31      2008               911  WORLDBANKINCOMEGROUP  Low-income economies   
32      2009               911  WORLDBANKINCOMEGROUP  Low-income economies   
33      2010               911  WORLDBANKINCOMEGROUP  Low-income economies   
34      2011               911  WORLDBANKINCOMEGROUP  Low-income economies   
35      2012               911  WORLDBANKINCOMEGROUP  Low-income economies   
36      2013               911  WORLDBANKINCOMEGROUP  Low-income economies   
37      2014               911  WORLDBANKINCOMEGROUP  Low-income economies   
38      2015               911  WORLDBANKINCOMEGROUP  Low-income economies   
39      2016               911  WORLDBANKINCOMEGROUP  Low-income economies   
40      2017               911  WORLDBANKINCOMEGROUP  Low-income economies   
41      2018               911  WORLDBANKINCOMEGROUP  Low-income economies   
42      2019               911  WORLDBANKINCOMEGROUP  Low-income economies   
43      2020               911  WORLDBANKINCOMEGROUP  Low-income economies   
44      2021               911  WORLDBANKINCOMEGROUP  Low-income economies   
45      2022               911  WORLDBANKINCOMEGROUP  Low-income economies   

    RATE_PER_100000_N  RATE_PER_100000_NL  RATE_PER_100000_NU  
23              274.0               185.0               363.0  
24              269.0               185.0               353.0  
25              264.0               186.0               343.0  
26              260.0               187.0               333.0  
27              255.0               187.0               322.0  
28              249.0               187.0               312.0  
29              245.0               187.0               302.0  
30              240.0               186.0               294.0  
31              233.0               184.0               282.0  
32              229.0               183.0               275.0  
33              224.0               182.0               267.0  
34              221.0               188.0               255.0  
35              217.0               182.0               252.0  
36              213.0               180.0               245.0  
37              208.0               178.0               238.0  
38              203.0               178.0               228.0  
39              199.0               174.0               224.0  
40              195.0               171.0               219.0  
41              191.0               168.0               214.0  
42              188.0               165.0               210.0  
43              184.0               162.0               206.0  
44              181.0               159.0               203.0  
45              181.0               159.0               204.0

What is the incidence trend for High income countries?

The incidence trend over the period 2000 to 2022 can be acquired by using the .loc function in the GEO_NAME_SHORT column to look for 'High-income-countries'. The incidence drops from an average of 21 to 8.9 cases per 100,000 people.

In [50]:
Income.loc[Income['GEO_NAME_SHORT'] == 'High-income economies']

DIM_TIME  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE         GEO_NAME_SHORT  \
0       2000               910  WORLDBANKINCOMEGROUP  High-income economies   
1       2001               910  WORLDBANKINCOMEGROUP  High-income economies   
2       2002               910  WORLDBANKINCOMEGROUP  High-income economies   
3       2003               910  WORLDBANKINCOMEGROUP  High-income economies   
4       2004               910  WORLDBANKINCOMEGROUP  High-income economies   
5       2005               910  WORLDBANKINCOMEGROUP  High-income economies   
6       2006               910  WORLDBANKINCOMEGROUP  High-income economies   
7       2007               910  WORLDBANKINCOMEGROUP  High-income economies   
8       2008               910  WORLDBANKINCOMEGROUP  High-income economies   
9       2009               910  WORLDBANKINCOMEGROUP  High-income economies   
10      2010               910  WORLDBANKINCOMEGROUP  High-income economies   
11      2011               910  WORLDBANKINCOMEGROUP  High-income economies   
12      2012               910  WORLDBANKINCOMEGROUP  High-income economies   
13      2013               910  WORLDBANKINCOMEGROUP  High-income economies   
14      2014               910  WORLDBANKINCOMEGROUP  High-income economies   
15      2015               910  WORLDBANKINCOMEGROUP  High-income economies   
16      2016               910  WORLDBANKINCOMEGROUP  High-income economies   
17      2017               910  WORLDBANKINCOMEGROUP  High-income economies   
18      2018               910  WORLDBANKINCOMEGROUP  High-income economies   
19      2019               910  WORLDBANKINCOMEGROUP  High-income economies   
20      2020               910  WORLDBANKINCOMEGROUP  High-income economies   
21      2021               910  WORLDBANKINCOMEGROUP  High-income economies   
22      2022               910  WORLDBANKINCOMEGROUP  High-income economies   

    RATE_PER_100000_N  RATE_PER_100000_NL  RATE_PER_100000_NU  
0                21.0                20.0                22.0  
1                21.0                21.0                22.0  
2                21.0                20.0                22.0  
3                21.0                20.0                22.0  
4                19.0                18.0                20.0  
5                19.0                19.0                20.0  
6                19.0                18.0                19.0  
7                18.0                17.0                18.0  
8                18.0                17.0                18.0  
9                17.0                16.0                18.0  
10               17.0                16.0                17.0  
11               16.0                16.0                17.0  
12               15.0                15.0                16.0  
13               15.0                14.0                15.0  
14               14.0                14.0                15.0  
15               14.0                13.0                14.0  
16               13.0                13.0                14.0  
17               13.0                12.0                13.0  
18               12.0                12.0                12.0  
19               11.0                11.0                12.0  
20                9.2                 8.9                 9.5  
21                9.1                 8.9                 9.4  
22                8.9                 8.6                 9.2

What is the incidence trend for Lower middle income countries?

The incidence trend over the period 2000 to 2022 can be acquired by using the .loc function in the GEO_NAME_SHORT column to look for 'Lower-middle-income-countries'. The incidence drops from an average of 291 to 206 cases per 100,000 people.

In [53]:
Income.loc[Income['GEO_NAME_SHORT'] == 'Lower-middle-income economies']

DIM_TIME  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE  \
46      2000               912  WORLDBANKINCOMEGROUP   
47      2001               912  WORLDBANKINCOMEGROUP   
48      2002               912  WORLDBANKINCOMEGROUP   
49      2003               912  WORLDBANKINCOMEGROUP   
50      2004               912  WORLDBANKINCOMEGROUP   
51      2005               912  WORLDBANKINCOMEGROUP   
52      2006               912  WORLDBANKINCOMEGROUP   
53      2007               912  WORLDBANKINCOMEGROUP   
54      2008               912  WORLDBANKINCOMEGROUP   
55      2009               912  WORLDBANKINCOMEGROUP   
56      2010               912  WORLDBANKINCOMEGROUP   
57      2011               912  WORLDBANKINCOMEGROUP   
58      2012               912  WORLDBANKINCOMEGROUP   
59      2013               912  WORLDBANKINCOMEGROUP   
60      2014               912  WORLDBANKINCOMEGROUP   
61      2015               912  WORLDBANKINCOMEGROUP   
62      2016               912  WORLDBANKINCOMEGROUP   
63      2017               912  WORLDBANKINCOMEGROUP   
64      2018               912  WORLDBANKINCOMEGROUP   
65      2019               912  WORLDBANKINCOMEGROUP   
66      2020               912  WORLDBANKINCOMEGROUP   
67      2021               912  WORLDBANKINCOMEGROUP   
68      2022               912  WORLDBANKINCOMEGROUP   

                   GEO_NAME_SHORT  RATE_PER_100000_N  RATE_PER_100000_NL  \
46  Lower-middle-income economies              291.0               166.0   
47  Lower-middle-income economies              290.0               166.0   
48  Lower-middle-income economies              290.0               167.0   
49  Lower-middle-income economies              289.0               167.0   
50  Lower-middle-income economies              287.0               167.0   
51  Lower-middle-income economies              284.0               167.0   
52  Lower-middle-income economies              280.0               168.0   
53  Lower-middle-income economies              275.0               167.0   
54  Lower-middle-income economies              269.0               167.0   
55  Lower-middle-income economies              264.0               167.0   
56  Lower-middle-income economies              258.0               167.0   
57  Lower-middle-income economies              252.0               182.0   
58  Lower-middle-income economies              245.0               182.0   
59  Lower-middle-income economies              238.0               181.0   
60  Lower-middle-income economies              233.0               194.0   
61  Lower-middle-income economies              228.0               197.0   
62  Lower-middle-income economies              220.0               198.0   
63  Lower-middle-income economies              214.0               196.0   
64  Lower-middle-income economies              208.0               191.0   
65  Lower-middle-income economies              204.0               186.0   
66  Lower-middle-income economies              198.0               179.0   
67  Lower-middle-income economies              203.0               183.0   
68  Lower-middle-income economies              206.0               184.0   

    RATE_PER_100000_NU  
46               416.0  
47               415.0  
48               413.0  
49               411.0  
50               406.0  
51               400.0  
52               392.0  
53               382.0  
54               371.0  
55               361.0  
56               349.0  
57               323.0  
58               308.0  
59               295.0  
60               272.0  
61               259.0  
62               242.0  
63               232.0  
64               226.0  
65               221.0  
66               217.0  
67               223.0  
68               227.0

What is the incidence trend for Upper middle income countries?

The incidence trend over the period 2000 to 2022 can be acquired by using the .loc function in the GEO_NAME_SHORT column to look for 'Upper-middle-income-countries'. The incidence drops from an average of 134 to 95 cases per 100,000 people.

In [54]:
Income.loc[Income['GEO_NAME_SHORT'] == 'Upper-middle-income economies']

DIM_TIME  DIM_GEO_CODE_M49     DIM_GEO_CODE_TYPE  \
69      2000               914  WORLDBANKINCOMEGROUP   
70      2001               914  WORLDBANKINCOMEGROUP   
71      2002               914  WORLDBANKINCOMEGROUP   
72      2003               914  WORLDBANKINCOMEGROUP   
73      2004               914  WORLDBANKINCOMEGROUP   
74      2005               914  WORLDBANKINCOMEGROUP   
75      2006               914  WORLDBANKINCOMEGROUP   
76      2007               914  WORLDBANKINCOMEGROUP   
77      2008               914  WORLDBANKINCOMEGROUP   
78      2009               914  WORLDBANKINCOMEGROUP   
79      2010               914  WORLDBANKINCOMEGROUP   
80      2011               914  WORLDBANKINCOMEGROUP   
81      2012               914  WORLDBANKINCOMEGROUP   
82      2013               914  WORLDBANKINCOMEGROUP   
83      2014               914  WORLDBANKINCOMEGROUP   
84      2015               914  WORLDBANKINCOMEGROUP   
85      2016               914  WORLDBANKINCOMEGROUP   
86      2017               914  WORLDBANKINCOMEGROUP   
87      2018               914  WORLDBANKINCOMEGROUP   
88      2019               914  WORLDBANKINCOMEGROUP   
89      2020               914  WORLDBANKINCOMEGROUP   
90      2021               914  WORLDBANKINCOMEGROUP   
91      2022               914  WORLDBANKINCOMEGROUP   

                   GEO_NAME_SHORT  RATE_PER_100000_N  RATE_PER_100000_NL  \
69  Upper-middle-income economies              134.0               104.0   
70  Upper-middle-income economies              134.0               106.0   
71  Upper-middle-income economies              134.0               107.0   
72  Upper-middle-income economies              134.0               108.0   
73  Upper-middle-income economies              133.0               108.0   
74  Upper-middle-income economies              132.0               106.0   
75  Upper-middle-income economies              130.0               105.0   
76  Upper-middle-income economies              128.0               103.0   
77  Upper-middle-income economies              126.0               107.0   
78  Upper-middle-income economies              124.0               107.0   
79  Upper-middle-income economies              120.0               103.0   
80  Upper-middle-income economies              118.0               103.0   
81  Upper-middle-income economies              116.0                99.0   
82  Upper-middle-income economies              112.0                97.0   
83  Upper-middle-income economies              109.0                95.0   
84  Upper-middle-income economies              106.0                93.0   
85  Upper-middle-income economies              100.0                91.0   
86  Upper-middle-income economies               98.0                90.0   
87  Upper-middle-income economies               95.0                88.0   
88  Upper-middle-income economies               92.0                86.0   
89  Upper-middle-income economies               90.0                84.0   
90  Upper-middle-income economies               91.0                85.0   
91  Upper-middle-income economies               95.0                87.0   

    RATE_PER_100000_NU  
69               164.0  
70               162.0  
71               160.0  
72               159.0  
73               159.0  
74               157.0  
75               155.0  
76               153.0  
77               145.0  
78               142.0  
79               137.0  
80               133.0  
81               133.0  
82               127.0  
83               123.0  
84               119.0  
85               110.0  
86               107.0  
87               103.0  
88                99.0  
89                96.0  
90                98.0  
91               102.0